modules/epomDspBidAdapter.md

```diff
@@ -0,0 +1,166 @@
+
+# Overview
+
+```
+Module Name:  Epom DSP Bid Adapter
+Module Type:  Bidder Adapter
+Maintainer: support@epom.com
+```
+
+# Description
+
+The **Epom DSP Bid Adapter** connects publishers to the Epom DSP Exchange for programmatic advertising. This adapter supports banner formats and follows the OpenRTB protocol.
+
+# Supported Media Types
+
+- **Banner**
+
```

Comment:
Please explicitly list the rest of the supported fields for your adapter, see other bidder md files for examples. this same info will be used in your prebid website PR

```
tcf2_supported: true
media_types: display
floors_supported: true

Reply:
added